src/renderer/widgets/CreateWallet/model/form-model.ts

```diff
@@ -132,6 +127,44 @@ const $availableAccounts = combine(
   },
 );
 
+const $invalidAddresses = combine(
+  {
+    chain: $chain,
+    signatories: signatoryModel.$signatories,
+  },
+  ({ chain, signatories }) => {
+    if (!chain) return new Set();
+
+    const badSignatories: Set<Address> = new Set();
+
+    for (const signer of signatories) {
+      if (!signer.address || validateAddress(signer.address, chain)) continue;
+
+      badSignatories.add(signer.address);
+    }
+
+    return badSignatories;
+  },
+);
+
+const $canSubmit = combine(
+  {
+    hasEmptySignatories: signatoryModel.$hasEmptySignatories,
+    hasEmptySignatoryName: signatoryModel.$hasEmptySignatoryName,
+    hasDuplicateSignatories: signatoryModel.$hasDuplicateSignatories,
+    multisigAlreadyExists: $multisigAlreadyExists,
+    invalidAddresses: $invalidAddresses,
+    hiddenMultisig: $hiddenMultisig,
+    threshold: $createMultisigForm.fields.threshold.$value,
+  },
+  ({ invalidAddresses, threshold, ...params }) => {
+    if (invalidAddresses.size > 0) return false;
+    if (threshold < MIN_THRESHOLD) return false;
```

Comment:
What is the reason to divide it? Maybe better create variables and use `||` as we do mostly?